### PR TITLE
Fix/campaigns comments

### PR DESCRIPTION
**Pull request by gmzjuliana**
### Purpose
fixes the following campaigns bugs:
- Wonder if we can line height the text so it can accommodate emojis more nicely [[Ref]](https://share.buffer.com/E0uq1GAX)
- Posts Metrics selector buttons are clickable in reports [[ref]](https://share.buffer.com/jkuK5gNb)
- For Avg. Engagement Rate under Campaign posts metrics, the tooltip reads “The average number of engagements on your posts”. Should it be “The average engagement rate of all your posts in this campaign”? 
- Just noticed this, part of the new tweaks to the Post Breakdown table the height of the rows are actually smaller by around 30-40px I think. We can squeeze some more space there by making those Post Breakdown rows to now be less in height. According to the designs they’re now 200px high maximum [[Ref]](https://share.buffer.com/rRu6Y8D6)
- The CSV export doesn’t contain the campaign name that was exported. Only lists “campaign-pulse” [[ref]](https://share.buffer.com/wbumEv7r) ** made the whole export folder include the campaign name instead of individually on each file **
- The tooltip for engagement rate should have a %, and say average engagement rate instead of average_engagement_rate [[ref]](https://share.buffer.com/L1ugmWe5) 
- The tooltip for engagement should read engagements not engagement and not have a % [[ref]](https://share.buffer.com/kpuYg4AQ)

#### Staging Deployment

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.analyze.buffer.com :smile:_

### CODE REVIEW

**Comment by gmzjuliana** [comment]
hey @moreofmorris could I get your eyes on:

- Wonder if we can line height the text so it can accommodate emojis more nicely [Ref]
- Just noticed this, part of the new tweaks to the Post Breakdown table the height of the rows are actually smaller by around 30-40px I think. We can squeeze some more space there by making those Post Breakdown rows to now be less in height. According to the designs they’re now 200px high maximum [Ref]

thanks!

**Comment by moreofmorris** [comment]
Thank you @gmzjuliana, looks great! ❤️

File: packages/campaign-posts-metrics/components/ChartTooltip/index.jsx [hunk 1]
```diff
@@ -47,24 +47,30 @@ const TextWrapper = styled.div`
...
-  font-weight: ${props => (props.normal ? "400" : "500")};
+  font-weight: ${(props) => (props.normal ? "400" : "500")};
   font-size: 14px;
   line-height: 16px;
   padding-bottom: 8px;
   color: white;
 `;
 
 export const ChartTooltip = ({ day, data, selectedMetric }) => {
+  selectedMetric =
```
Comment (federicoweber) [inline packages/campaign-posts-metrics/components/ChartTooltip/index.jsx:58]: let's wrap this into a `formatMetricLabel`, with a switch statement to decide which metric to transform 🙂.

File: packages/campaigns/components/Header/index.jsx [hunk 2]
```diff
@@ -116,52 +116,74 @@ const ExportPickerTempStyle = styled.div`
   }
 `;
 
+const removeEmoji = (name) => {
```
Comment (federicoweber) [inline packages/campaigns/components/Header/index.jsx:119]: great call 🙌

**Pull request merged**